repository/src/main/java/org/apache/atlas/repository/graph/GraphHelper.java

```diff
@@ -455,6 +455,26 @@ public static List<AtlasEdge> getPropagatedEdges(AtlasVertex classificationVerte
         return ret;
     }
 
+    public static List<AtlasVertex> getPropagatedVertices (AtlasVertex classificationVertex) {
+        List<AtlasVertex>   ret      =  new ArrayList<>();
+        Iterable            vertices =  classificationVertex.query().direction(AtlasEdgeDirection.IN).label(CLASSIFICATION_LABEL)
+                                                            .has(CLASSIFICATION_EDGE_IS_PROPAGATED_PROPERTY_KEY, true)
+                                                            .has(CLASSIFICATION_EDGE_NAME_PROPERTY_KEY, getTypeName(classificationVertex))
+                                                            .vertices();
+
+        if (vertices != null) {
+            Iterator<AtlasVertex> iterator = vertices.iterator();
```

Comment:
better way to convert Iterable to list? https://www.baeldung.com/java-iterable-to-collection